compiler/rustc_passes/src/check_attr.rs

```diff
@@ -768,7 +768,7 @@ impl CheckAttrVisitor<'tcx> {
                             "not a `use` item",
                         );
                     }
-                    err.note("read https://doc.rust-lang.org/nightly/rustdoc/the-doc-attribute.html#docno_inlinedocinline for more information")
+                    err.note("read https://doc.rust-lang.org/nightly/rustdoc/the-doc-attribute.html#inline_and_no_inline for more information")
```

Comment:
I think this link is not quite correct. It should be:

```suggestion
                    err.note("read https://doc.rust-lang.org/nightly/rustdoc/the-doc-attribute.html#inline-and-no_inline for more information")
```

Reply:
Arf, copied the link I created and forgot to regen the book... Great catch!